### PR TITLE
Add Свободу байтам

### PR DESCRIPTION
**Pull request by dzianisv**

### CODE REVIEW

File: articles/Свободу байтам.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+"Как так? Нам же говорили, что сеть Интернет открыта и свободна". Чисто гипотетически, представьте себе ситуацию, что власть захватили вражеские агенты, мы в оккупации, нас помещают в информационный пузырь.
+
+> Вы видите меня хозяином Франции, но я бы не взялся править ею и три месяца при свободной прессе.
+>
+> — [Наполеон I](https://www.forbes.ru/forbeslife/obrazovanie/241205-22-luchshie-tsitaty-o-svobodnoi-presse)
+
+
+Конечно гипотетически, на самом деле такое с нами произойти не может. А то, что сейчас блокируют (как устроена [блокировка в РФ](https://habr.com/ru/company/cloud4y/blog/347964/)), это же конечно "для нашего блага".
+Я собрал в статье все открытые или, как минимум, *закрытые но бесплатные и удобные* инструменты, которые позволяют получить доступ к информации в случаее ее частичной или полной блокировки доступа в Сеть интернет. На всякий случай.
```
Comment (stablebits) [inline articles/Свободу байтам.md:11]: > доступ к информации в случае**е** **ее** частичной или полной блокировки доступа в Сеть интернет

drop "ее" ?

"Сеть интернет" -> "Iнтернет"?

File: articles/Свободу байтам.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+<cut />
+
+> Disclaimer: вдохновлением для этой статьи послужил беспредел, который происходит в родных краях. Допускаю, что о всех рассмотренных проблемах и решениях уже писали. Статья носит характер агрегации доступных мне способо решения проблем блокировки к Интернет ресурсам.
```
Comment (stablebits) [inline articles/Свободу байтам.md:15]: heh, bespredel skoree pobujdaet chem vdohnovliaet. hotia esli etim peredaetsia ironia ))

Comment (stablebits) [inline articles/Свободу байтам.md:15]: > Статья носит характер агрегации доступных мне **способо** решения проблем блокировки к Интернет ресурсам

kak-to stranno zvuchit

File: articles/Свободу байтам.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Все виды блокировок можно обойти с помощью VPN. Но могут заблокировать сам VPN. Для таких случаев придумали Tor.
+Решения расположены в порядке простоты их использования.
+
+## [WARP+](https://1.1.1.1/)
+
+Закрытый бесплатный VPN от [Cloudflare](https://blog.cloudflare.com/announcing-warp-plus/) на основе протокола [Wireguard](https://www.wireguard.com/). Рекламы нет, поэтому поместил в этот список.
+
+![1.1.1.1 app](https://lh3.googleusercontent.com/FtmQWtaroA4BAXAK4evPMjyhFZA_f_Y3nkeXIOTqkpJczjK6oH6xzkxy1qZ60YN5AEM=w3318-h1674)
+
+Так как используется Wireguard, который поддерживает [роуминг](https://www.wireguard.com/#built-in-roaming), то смена сетей (Wi-Fi, LTE, 3G) происходит без переподключения, и, если вы загружали какой-то файл или видео, то загрузка не оборвется.
```
Comment (stablebits) [inline articles/Свободу байтам.md:45]: sort of irrelevant to anti-blocking?

File: articles/Свободу байтам.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+Cписок бесплатных OpenVPN серверов, которые хостятся в основном США и Японии.
+Будет работать с любым OpenVPN клиентом на любой операционной системе.
+
+- 👎 Протокол блокируется Великим Китайским Фаерволом.
++ 👍 OpenVPN клиенты есть для всех популярных операционных систем.
+- 👎 Тяжело в использовании неопытными пользователями, необходим скачивать конфигурацию самостоятельно. Нужно скачать конфигурационный файл с сайта и выбрать его в приложении:  ,
+
+## Вывод
+Это сложнее, но дешевле (от 3 евро в месяц), если VPN вы собираетесь пользоваться много. Для просмотра нескольких ресурсов хватит того, о чем я написал выше. Я также не рассматривал такие продукты, как TunnelBear (на базе [OpenVPN, IPSec](https://www.security.org/vpn/tunnelbear/review/)), NordVPN (на базе [OpenVPN](https://www.security.org/vpn/nordvpn/review/)) и т.п.. Потому что они платные, а в бесплатных версиях - навязчивая реклама.
```
Comment (stablebits) [inline articles/Свободу байтам.md:81]: but you mentioned "from 3 euro" above when describing "free" products

File: articles/Свободу байтам.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+- 👍 [Obfs4](https://blog.torproject.org/learning-more-about-gfws-active-probing-system) позволяет обходить даже Великий Китайский Фаервол.
+- 👍 [Открытый код](https://github.com/shadowsocks).
+- 👎 UX для опытных пользоавтелей.
+- 👎 Публичных бесплатных серверов я не видел. Но можно очень просто развернуть свой сервер на [Linode](https://www.linode.com/docs/platform/one-click/deploy-shadowsocks-with-one-click-apps/).
+- 👍 Поддерживается на всех популярных операционных системах, в том числе на мобильных.
+
+
+## Выводы
+
+Выбор редакции  - Tor. Tor поможет в *большинстве случаев*. Но если с ним возиться вам не охота, то стоит все же присмотреться к [Hotspotshield](https://www.hotspotshield.com/), который использует пропиретарный протокол [Catapult Hydra](https://support.hotspotshield.com/hc/en-us/articles/360000374343) и маскируется под HTTPS веб-трафик.I still don't understand your viewpoint that everything that's not under your umbrella is hostile. Never at any point was I hostile in discussions with you. You should be happy that people embrace your fantastic protocol instead of having a negative attitude.
```
Comment (stablebits) [inline articles/Свободу байтам.md:124]: wtf is this? also, are you sure that hotspotshield even works in China? I'm not actually.

Comment (stablebits) [reply articles/Свободу байтам.md:124]: and here, if you post under your real name (?), someone finds out that you are (or were) affiliated with the company behind HSS :D

File: articles/Свободу байтам.md (hunk 3 above)
Comment (dzianisv) [reply articles/Свободу байтам.md:45]: it's ok

File: articles/Свободу байтам.md (hunk 5 above)
Comment (dzianisv) [reply articles/Свободу байтам.md:124]: I am not affiliated, but I know just one commercial product that can bypass GCF https://www.hotspotshield.com/blog/china-travel-tips-vpn/. I, even more, have written about Cloudflare products (WARP+, 1.1.1.1, DoH) here. But I am not affiliated with them too.

**Pull request merged**